src/functions/asyncToken/requestService/requestService.ts

```diff
@@ -17,8 +16,9 @@ export class RequestService implements IProcessRequest {
       return log("Invalid authorization header");
     }
 
-    const decodeAuthorizationHeader =
-      this.decodeAuthorizationHeader(authorizationHeader);
+    const decodeAuthorizationHeader = this.decodeAuthorizationHeader(
+      authorizationHeader!,
```

Comment:
We could avoid having to use `!` if we take out the validation from the private function:
```
    if (!authorizationHeader) {
      return log("Missing authorization header");
    }

    if (!authorizationHeader.startsWith("Basic ")) {
      return log("Invalid authorization header");
    }
```
But confident with it remaining as is